entrypoint.sh

```diff
@@ -86,7 +102,7 @@ else
             testargs+=("-run" "${TEST_RUN}")
         fi
 
-        go test -v "${testargs[@]}" "./$pkg"
+        go test -v ${BUILD_TAGS} "${testargs[@]}" "./$pkg"
```

Comment:
Please combine the needed tags with the testargs.

Reply:
Done!